test/pubsub-memory.js

```diff
@@ -13,7 +13,7 @@ describe('PubSub base class', function() {
   it('returns an error if _subscribe is unimplemented', function(done) {
     var pubsub = new PubSub();
     pubsub.subscribe('x', function(err) {
-      expect(err).an(Error);
+      expect(err).ok;
```

Comment:
For this file, Chai equivalent of the old `expect(err).an(Error)` would be `expect(err).to.be.an('Error')`.